solr/contrib/ltr/src/test/org/apache/solr/ltr/TestParallelWeightCreation.java

```diff
@@ -70,7 +70,7 @@ public void testLTRQParserThreadInitialization() throws Exception {
     // set maxThreads to 1 and maxQueryThreads to 2 and verify that an exception is thrown
     String msg2 = null;
     try{
-      new LTRThreadModule(1,2);
+      new LTRThreadModule(1,2).validate();
```

Comment:
This seems odd that you have to manually call validate now after constructing the class.  The class's constructor should automatically validate itself for you.  It will be very very easy to forget to call this function somewhere else.